packages/env-utilities/src/lib/types.ts

```diff
@@ -0,0 +1,18 @@
+export type BooleanString = 'true' | 'false';
+export type IntegerString = `${bigint}`;
```

Comment:
why bigint? That's pretty nonsensical for env vars. Not to mention that in the eventual code you don't use bigint anyway but rather Number.

Reply:
`${bigint}` denotes any integer string, such as Discord snowflakes. `${number}` allows for decimal strings such as `1.1`.

Reply:
`${bigint}` denotes any integer string, such as Discord snowflakes. `${number}` allows for decimal strings such as `1.1`.